_cxx/envoy.mk

```diff
@@ -273,6 +273,7 @@ $(OSS_HOME)/pkg/api/envoy: $(OSS_HOME)/pkg/api/%: $(OSS_HOME)/_cxx/envoy/build_g
 	      -- {} +; \
 	  find "$$tmpdir" -name '*.bak' -delete; \
 	  mv "$$tmpdir/$*" $@; \
+	  gofmt -w -s ./pkg/api/envoy; \
```

Comment:
Isn't `./pkg/api/envoy` generated code? I'm guessing the envoy build container hasn't been updated to 1.19 so its probably not formatting in 1.19 doc format?

In general, I guess I would have thought that we would exclude running gofmt on generated code so just curious on this one?

Reply:
alright thanks for the explanation. I'm cool with this.